### PR TITLE
Point download to latest release version + local.properties update.

### LINKED ISSUE #37: Add note about releases and how to download to readme

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #37.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,15 +63,11 @@ See [instruction](HOW_TO_SETUP_REMOTE_BUILD_MACHINE.md). It’s actually very ea
...
 
-1. Download [`mainframer.sh`](mainframer.sh) and save it in your project, most likely put it to VCS so you could sync changes across all team members.
+1. **Download** [latest release version of `mainframer.sh`](https://github.com/gojuno/mainframer/releases/latest) and save it in your project, most likely put it to VCS so you could sync changes across all team members.
 
   We recommend you to subscribe to changes in this repo somehow (follow it on GitHub / watch for tweets of its maintainers / etc). This will allow you to apply best practises we found to make your Remote Build better, faster and safer.
 
-2. Update your `local.properties` file.
-
-  ```
-  $ echo "remote_build.machine={BUILD_MACHINE_NAME}" >> local.properties
```
Comment (artem-zinnatullin) [inline README.md]: This was wrong because it adds property to last line, which might break file parsing.

Comment (ming13) [inline README.md]: User can (and most likely will) add it as a last line. Don’t see the difference between changes.

File: README.md [hunk 2]
```diff
@@ -63,15 +63,11 @@ See [instruction](HOW_TO_SETUP_REMOTE_BUILD_MACHINE.md). It’s actually very ea
 
 # Local Configuration
 
-1. Download [`mainframer.sh`](mainframer.sh) and save it in your project, most likely put it to VCS so you could sync changes across all team members.
+1. **Download** [latest release version of `mainframer.sh`](https://github.com/gojuno/mainframer/releases/latest) and save it in your project, most likely put it to VCS so you could sync changes across all team members.
```
Comment (ming13) [inline README.md]: Bold is unnecessary actually.

File: README.md (hunk 1 above)
Comment (artem-zinnatullin) [inline README.md]: No, if last line is already a property declaration it'll squash them

File: README.md (hunk 2 above)
Comment (artem-zinnatullin) [inline README.md]: Wanted to highlight that

**Comment by artem-zinnatullin** [comment]
Nothing to fix imho :)

File: README.md [hunk 3]
```diff
@@ -63,15 +63,11 @@ See [instruction](HOW_TO_SETUP_REMOTE_BUILD_MACHINE.md). It’s actually very ea
...
 
   We recommend you to subscribe to changes in this repo somehow (follow it on GitHub / watch for tweets of its maintainers / etc). This will allow you to apply best practises we found to make your Remote Build better, faster and safer.
 
-2. Update your `local.properties` file.
-
-  ```
-  $ echo "remote_build.machine={BUILD_MACHINE_NAME}" >> local.properties
-  ```
+2. Add `remote_build.machine={BUILD_MACHINE_NAME}` to `local.properties`.
```
Comment (ming13) [inline README.md]: Add the following content to `local.properties`.

```properties
remote_build.machine={BUILD_MACHINE_NAME}
```

Comment (artem-zinnatullin) [inline README.md]: Foxed

**Pull request merged**